### PR TITLE
Feat(automation locators)

### PR DESCRIPTION
**Pull request by TiManGames**
Automation locators were added to almost every application component (`test-id` attributes) in order to have a simpler and easier automated testing development. 

**This is very crucial for us so we can have automation developed quickly.**

- Added ids to ExposureForm
- Added ids to FlightsForm
- Added ids to ExposuresAndFlights
- Added optional id field to interface FormRowWithInputProps 
- Added optional id property to InfoItem
- Added optional id property to InfoWithIcon
- Added ids to InvestigationMetadata
- Added ids to InvestigationForm
- Added optional id property to CustomCheckbox
- Added ids to PersonalInfoTab
- Added Ids to clinical details 
- Added IDs to ContactDateCard
- Added IDs to InteractionEventForm
- Added IDs to NewInteractionEventDialog

### CODE REVIEW

**Review by MayaSasson — DISMISSED** [review]

**Review by eladyashar — DISMISSED** [review]

File: client/src/commons/CheckBox/CustomCheckbox.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { AriaAttributes, DOMAttributes } from 'react';
```
Comment (RoeiRom) [inline client/src/commons/CheckBox/CustomCheckbox.tsx:1]: where do you use this?

File: client/src/commons/CheckBox/CustomCheckbox.tsx [hunk 2]
```diff
@@ -32,9 +33,11 @@ export default CustomCheckbox;
 
 interface CheckboxElement extends CheckboxProps {
     labelText: string | JSX.Element;
+    'test-id'?: string;
```
Comment (RoeiRom) [inline client/src/commons/CheckBox/CustomCheckbox.tsx:36]: is this needed

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx [hunk 3]
```diff
@@ -26,12 +26,10 @@ const NewInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Element
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
 
-    const canConfirm = React.useMemo<boolean>(() => true, [])
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx:29]: why deleting it?

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx [hunk 4]
```diff
@@ -51,15 +49,15 @@ const NewInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Element
...
-                <Button 
+                <Button
+                    test-id='cancelNewContactLocation'
                     onClick={() => closeDialog()}
                     color='default' 
                     className={classes.cancelButton}>
                     בטל
                 </Button>
-                <PrimaryButton 
-                    disabled={!canConfirm}
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx:61]: also this

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx (hunk 4 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx:61]: I did not tamper with this line, not sure how it got deleted. This should be restored.

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx (hunk 3 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx:29]: I did not tamper with this line, not sure how it got deleted. This should be restored.

File: client/src/commons/CheckBox/CustomCheckbox.tsx (hunk 2 above)
Comment (TiManGames) [reply client/src/commons/CheckBox/CustomCheckbox.tsx:36]: Yes, it's so we can add a `test-id` attribute to checkboxes.

File: client/src/commons/CheckBox/CustomCheckbox.tsx (hunk 1 above)
Comment (TiManGames) [reply client/src/commons/CheckBox/CustomCheckbox.tsx:1]: I used this previously, then reverted the changes and forgot to remove the import.

Comment (RoeiRom) [reply client/src/commons/CheckBox/CustomCheckbox.tsx:1]: so remove

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx (hunk 4 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx:61]: Done.

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx (hunk 3 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx:29]: Done.

File: client/src/commons/CheckBox/CustomCheckbox.tsx (hunk 1 above)
Comment (TiManGames) [reply client/src/commons/CheckBox/CustomCheckbox.tsx]: Done.

**Review by RoeiRom — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]

**Pull request merged**